README.md

```diff
@@ -59,6 +59,13 @@ typeGuards.normandy_checkConsoleLogArguments({ message: "It works!" });
  */
 ```
 
+Shared data can be access by importing `data` from the package:
+
+````js
```

Comment:
Why the four `` ` `` for the code fence?

Reply:
Oops